types/meteor/mongo.d.ts

```diff
@@ -27,174 +27,314 @@ declare module "meteor/mongo" {
-            $gt?: T | undefined,
-            $gte?: T | undefined,
-            $lt?: T | undefined,
-            $lte?: T | undefined,
-            $in?: T[] | undefined,
-            $nin?: T[] | undefined,
-            $ne?: T | undefined,
-            $exists?: boolean | undefined,
-            $type?: BsonType[] | BsonType | undefined,
-            $not?: FieldExpression<T> | undefined,
-            $expr?: FieldExpression<T> | undefined,
-            $jsonSchema?: any,
-            $mod?: number[] | undefined,
-            $regex?: RegExp | string | undefined,
-            $options?: string | undefined,
-            $text?: { $search: string, $language?: string | undefined, $caseSensitive?: boolean | undefined, $diacriticSensitive?: boolean | undefined } | undefined,
-            $where?: string | Function | undefined,
-            $geoIntersects?: any,
-            $geoWithin?: any,
-            $near?: any,
-            $nearSphere?: any,
-            $all?: T[] | undefined,
-            $elemMatch?: T extends {} ? Query<T> : FieldExpression<T> | undefined | undefined | undefined | undefined | undefined | undefined | undefined | undefined,
-            $size?: number | undefined,
-            $bitsAllClear?: any,
-            $bitsAllSet?: any,
-            $bitsAnyClear?: any,
-            $bitsAnySet?: any,
-            $comment?: string | undefined
-        }
+            $eq?: T | undefined;
+            $gt?: T | undefined;
+            $gte?: T | undefined;
+            $lt?: T | undefined;
+            $lte?: T | undefined;
+            $in?: T[] | undefined;
+            $nin?: T[] | undefined;
+            $ne?: T | undefined;
+            $exists?: boolean | undefined;
+            $type?: BsonType[] | BsonType | undefined;
+            $not?: FieldExpression<T> | undefined;
+            $expr?: FieldExpression<T> | undefined;
+            $jsonSchema?: any;
+            $mod?: number[] | undefined;
+            $regex?: RegExp | string | undefined;
+            $options?: string | undefined;
+            $text?:
+                | {
+                      $search: string;
+                      $language?: string | undefined;
+                      $caseSensitive?: boolean | undefined;
+                      $diacriticSensitive?: boolean | undefined;
+                  }
+                | undefined;
+            $where?: string | Function | undefined;
+            $geoIntersects?: any;
+            $geoWithin?: any;
+            $near?: any;
+            $nearSphere?: any;
+            $all?: T[] | undefined;
+            $elemMatch?: T extends {}
+                ? Query<T>
+                :
+                      | FieldExpression<T>
+                      | undefined
+                      | undefined
+                      | undefined
+                      | undefined
+                      | undefined
+                      | undefined
+                      | undefined
+                      | undefined;
```

Comment:
What happened here? I know you've just formatted the code, but maybe we could simplify it now?